source/css/_variables/base.styl

```diff
@@ -92,6 +92,7 @@ $font-weight-bolder       = 700
 
 // Font size
 $font-size-base           = 14px
```

Comment:
It seems duplicate here.

Reply:
Following the style of the other declaration, seems a safe "default" way in stylus. Isn't it?

Reply:
Yeah, but i think need to set 14px in config by default instead of reading `$font-size-base` var twice.
```yml
font:
  global:
    size: 14
```

Reply:
Well, I will create another PR then to get rid of this double declarations. I saw a lot of them during the patch making.

Reply:
Hope u know what u are doing.
Also, always remember what in NexT is 4 schemes. And if on one of them (for ex on Mist) worked well, on another scheme this can be broken. Muse and Mist have aproach structure, but Pisces is very difference from them. Gemini is mirror of Pisces with some style remakes and additions. So, please, make the tests at least on two schemes (Muse || Mist && Pisces || Gemini).